src/reference/v8.json

```diff
@@ -2791,7 +2791,7 @@
         "doc": "Determines whether an item exists in an array or a substring exists in a string.\n\n - [Measure distances](https://maplibre.org/maplibre-gl-js/docs/examples/measure/)",
         "example": {
           "syntax": {
-            "method": ["value", "value"],
+            "method": [["get", "value"], ["literal", "value"]],
```

Comment:
This is the syntax part of the example, so this won't be the right place to put it. 
You can use links to places in the description, for example add a page in the gl js examples with all forms of "in".
You can link to tests in this repo too.
Or your could extend the code that parses the v8 into docs to have multiple examples for expressions.